CMakeLists.txt

```
@@ -49,6 +49,13 @@ if(ROBOTOLOGY_USES_MATLAB)
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/template/startup_robotology_superbuild.m.in ${CMAKE_BINARY_DIR}/startup_robotology_superbuild.m)
 endif()
 
+if(ROBOTOLOGY_USES_PYTHON)
+  find_package(Python3 COMPONENTS Interpreter REQUIRED)
```

```suggestion
  find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
```

The development component is used by many of the included projects. As suggested in [FindPython3](https://cmake.org/cmake/help/v3.12/module/FindPython3.html#module:FindPython3), searching all the components together ensures version consistency.